packages/shared/src/chunker.ts

```diff
@@ -0,0 +1,49 @@
+// No clue why, but 3600 matches 4kb in the browser
+const MAX_CHUNK_SIZE = 3600;
+
+interface Chunk {
+	name: string;
+	value: string;
+}
+
+// Get cookie as chunks to set in the browser
+export function createChunks(key: string, value: string) {
+	// check the length and then split
+	const chunkCount = Math.ceil(value.length / MAX_CHUNK_SIZE);
+
+	if (chunkCount === 1) {
+		return value;
```

Comment:
I'd either skip it entirely and let it flow through the rest of code, or return `[value]` instead, this way we dont need to branch out to `Array.isArray` later on.